src/Bridges/Variable/single_bridge_optimizer.jl

```diff
@@ -0,0 +1,50 @@
+"""
+    SingleBridgeOptimizer{BT<:AbstractBridge, OT<:MOI.ModelLike} <: AbstractBridgeOptimizer
+
+The `SingleBridgeOptimizer` bridges any constrained variables supported by the
+bridge `BT`. This is in contrast with the [`MathOptInterface.Bridges.LazyBridgeOptimizer`](@ref)
+which only bridges the constrained variables that are unsupported by the internal model,
+even if they are supported by one of its bridges.
+
+!!! note
+    Two bridge optimizers using variable bridges cannot be used together as both
```

Comment:
What's the purpose of the `SingleBridgeOptimizer` given this restriction? Is it solely for testing?

Reply:
Yes mostly for testing, I have added a remark in `apimanual`